### ISSUE TITLE
pgfutter --help throws exception

### ISSUE THREAD

**Issue by 1tacocat1** [lukasmartinelli/pgfutter#48]
Downloaded and installed pgfutter binary on Mac (OSX 10.13.5) per readme.  When I run pgfutter --help I get the exception listed below.  I upgraded go from 1.10.2 -> 1.10.3 but it doesn't seem to help (I'm not sure if this is relevant)

```
failed MSpanList_Insert 0x5e4000 0x9a2e78b9883 0x0
fatal error: MSpanList_Insert

runtime stack:
runtime.throw(0x552fab)
	/goroot/src/runtime/panic.go:491 +0xad fp=0x7ffeefbff550 sp=0x7ffeefbff520
runtime.MSpanList_Insert(0x57bbe8, 0x5e4000)
	/goroot/src/runtime/mheap.c:692 +0x8f fp=0x7ffeefbff578 sp=0x7ffeefbff550
MHeap_FreeSpanLocked(0x5787e0, 0x5e4000, 0x100)
	/goroot/src/runtime/mheap.c:583 +0x163 fp=0x7ffeefbff5b8 sp=0x7ffeefbff578
MHeap_Grow(0x5787e0, 0x8, 0x0)
	/goroot/src/runtime/mheap.c:420 +0x1a8 fp=0x7ffeefbff5f8 sp=0x7ffeefbff5b8
MHeap_AllocSpanLocked(0x5787e0, 0x1, 0x0)
	/goroot/src/runtime/mheap.c:298 +0x365 fp=0x7ffeefbff638 sp=0x7ffeefbff5f8
mheap_alloc(0x5787e0, 0x1, 0x12, 0x0)
	/goroot/src/runtime/mheap.c:190 +0x121 fp=0x7ffeefbff660 sp=0x7ffeefbff638
runtime.MHeap_Alloc(0x5787e0, 0x1, 0x10000000012, 0x15159)
	/goroot/src/runtime/mheap.c:240 +0x66 fp=0x7ffeefbff698 sp=0x7ffeefbff660
MCentral_Grow(0x580558, 0x0)
	/goroot/src/runtime/mcentral.c:197 +0x8b fp=0x7ffeefbff700 sp=0x7ffeefbff698
runtime.MCentral_CacheSpan(0x580558, 0x0)
	/goroot/src/runtime/mcentral.c:85 +0x167 fp=0x7ffeefbff738 sp=0x7ffeefbff700
runtime.MCache_Refill(0x5e0000, 0x12, 0x0)
	/goroot/src/runtime/mcache.c:90 +0xa0 fp=0x7ffeefbff760 sp=0x7ffeefbff738
runtime.mcacheRefill_m()
	/goroot/src/runtime/malloc.c:368 +0x57 fp=0x7ffeefbff780 sp=0x7ffeefbff760
runtime.onM(0x440590)
	/goroot/src/runtime/asm_amd64.s:273 +0x9a fp=0x7ffeefbff788 sp=0x7ffeefbff780
runtime.mallocgc(0x120, 0x36db00, 0x0, 0x0)
	/goroot/src/runtime/malloc.go:178 +0x849 fp=0x7ffeefbff838 sp=0x7ffeefbff788
runtime.newobject(0x36db00, 0x5e0000)
	/goroot/src/runtime/malloc.go:353 +0x49 fp=0x7ffeefbff860 sp=0x7ffeefbff838
runtime.newG(0x2e27a)
	/goroot/src/runtime/proc.go:233 +0x2a fp=0x7ffeefbff878 sp=0x7ffeefbff860
allocg(0x5690a0)
	/goroot/src/runtime/proc.c:925 +0x1f fp=0x7ffeefbff888 sp=0x7ffeefbff878
runtime.malg(0x8000, 0x569320)
	/goroot/src/runtime/proc.c:2106 +0x1f fp=0x7ffeefbff8b8 sp=0x7ffeefbff888
runtime.mpreinit(0x569900)
	/goroot/src/runtime/os_darwin.c:137 +0x27 fp=0x7ffeefbff8d0 sp=0x7ffeefbff8b8
mcommoninit(0x569900)
	/goroot/src/runtime/proc.c:201 +0xc9 fp=0x7ffeefbff8f8 sp=0x7ffeefbff8d0
runtime.schedinit()
	/goroot/src/runtime/proc.c:138 +0x55 fp=0x7ffeefbff920 sp=0x7ffeefbff8f8
runtime.rt0_go(0x7ffeefbff950, 0x2, 0x7ffeefbff950, 0x0, 0x2, 0x7ffeefbffad0, 0x7ffeefbffad9, 0x0, 0x7ffeefbffae0, 0x7ffeefbffb1c, ...)
	/goroot/src/runtime/asm_amd64.s:95 +0x116 fp=0x7ffeefbff928 sp=0x7ffeefbff920
```

**Comment by dacoinminster** [comment]
I also had this problem, but installing from source as shown in the readme fixed it for me.

**Comment by 1tacocat1** [comment]
Thanks - installing from source worked for me as well.  Leaving open as it seems like the binary hosted should be updated (or just direct folks to install from source)

**Comment by lukasmartinelli** [comment]
Should be fixed with v1.2 https://github.com/lukasmartinelli/pgfutter/releases/tag/v1.2
